.github/workflows/python-demos.yml

```diff
@@ -65,4 +80,28 @@ jobs:
       run: |
         python3 -m venv .venv
         source .venv/bin/activate
-        python3 pv_speaker_demo.py --show_audio_devices
\ No newline at end of file
+        python3 pv_speaker_demo.py --show_audio_devices
+
+  build-windows-arm64:
+    runs-on: ${{ matrix.machine }}
+
+    strategy:
+      matrix:
+        machine: [pv-windows-arm64]
```

Comment:
is there a reason this can't be included with the self-hosted job? You write if statements in jobs if you need to exclude certain actions. If there isn't a lot of overlap, don't worry about it